pkgs/development/compilers/mrustc/bootstrap.nix

```diff
@@ -78,8 +72,8 @@ stdenv.mkDerivation rec {
   makeFlags = [
     # Use shared mrustc/minicargo/llvm instead of rebuilding them
     "MRUSTC=${mrustc}/bin/mrustc"
-    "MINICARGO=${mrustc-minicargo}/bin/minicargo"
-    "LLVM_CONFIG=${llvm_7.dev}/bin/llvm-config"
+    #"MINICARGO=${mrustc-minicargo}/bin/minicargo"  # FIXME: uncomment this so minicargo doesn't get rebuilt
```

Comment:
Do we want to uncomment it or not?

Reply:
nah, couldn't find how to make it work

Reply:
we should update the comment or get rid of it so that people can understand the difficulty of the TODO/FIXME